src/dashboard/src/components/archival_storage/tests/test_archival_storage.py

```diff
@@ -105,7 +109,7 @@ def dashboard_login_and_setup(client, django_user_model, username, password):
 def get_streaming_response(streaming_content):
     response_text = ""
     for response_char in streaming_content:
-        response_text = "{}{}".format(response_text, response_char.decode("utf8"))
+        response_text = "{}{}".format(response_text, response_char)
```

Comment:
Is this breaking functionality somehow at the moment? If I remember correctly I added it because it fails in Django 2.0 with Python 3 because the responses are binary.

Reply:
Suspect this was accidentally removed @replaceafill while i was wrestling with streaming content and missed in my comb-through. Will be sure to change it back.